packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKind.ts

```diff
@@ -30,6 +29,11 @@ export interface GenericChange {
      * Change to the node.
      */
     nodeChange: NodeChangeset;
+
+    /**
+     * The revision this change is part of. Undefined when it is implied from context.
```

Comment:
Does "from context" mean information "above" (i.e., you should have encountered a revision to get here) AND "below" (i.e., you'll encouter a revision further down), or just above? It would be good to clarify.